webapp/src/views/userSettings/accountSecurity/MfaSettings.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { FunctionComponent } from 'react';
+import { Box, Button, styled, Typography } from '@mui/material';
+import { T } from '@tolgee/react';
+import { Link } from 'react-router-dom';
+import { useUser } from 'tg.globalContext/helpers';
+import { LINKS } from 'tg.constants/links';
+
+const StyledEnabled = styled('span')`
+  color: ${({ theme }) => theme.palette.success.main};
+`;
+
+const StyledDisabled = styled('span')`
+  color: ${({ theme }) => theme.palette.error.main};
+`;
+
+const StyledEnabledParam = (value) => <StyledEnabled>{value}</StyledEnabled>;
+const StyledDisabledParam = (value) => <StyledDisabled>{value}</StyledDisabled>;
```

Comment:
You don't need these, it's possible to do it this way:
```
<T parameters={{ b: <StyledEnabled/> }}>
```
value will be injected as child automatically